src/templates/addons/terraformCloud/index.ts

```diff
@@ -0,0 +1,107 @@
+import dedent = require('dedent');
+import { prompt } from 'inquirer';
+
+import { GeneralOptions } from '@/commands/generate';
+import { injectToFile } from '@/helpers/file';
+import {
+  INFRA_BASE_MAIN_PATH,
+  INFRA_SHARED_MAIN_PATH,
+} from '@/templates/core/constants';
+
+type TerraformCloudOptions = {
+  terraformCloudEnabled: boolean;
+  terraformCloudOrganization: string;
+  terraformCloudBaseWorkspace: string;
+  terraformCloudSharedWorkspace: string;
+};
+
+const getTerraformCloudOptions = async (
+  projectName: string
+): Promise<TerraformCloudOptions> => {
+  const terraformCloudPrompt = await prompt([
+    {
+      type: 'confirm',
+      name: 'terraformCloudEnabled',
+      message:
+        'Would you like to enable Terraform Cloud?\nIf yes, the generated project will require 2 workspaces. One for the "shared" services, such as ERC, that will be used across all environments. The other workspace will be for the "base" services that will be created independently for each environment.',
+      default: false,
+    },
+  ]);
+
+  if (terraformCloudPrompt.terraformCloudEnabled) {
+    const terraformCloudOptions = await prompt([
+      {
+        type: 'input',
+        name: 'terraformCloudOrganization',
+        message: 'What is your Terraform Cloud organization?',
```

Comment:
```suggestion
        message: 'What is your Terraform Cloud organization name?',
```

(same for the 2 other questions below)

Reply:
Added in https://github.com/nimblehq/infrastructure-templates/pull/206/commits/1604e836acc362182ca1f8ea7de7830cfb3df4e4 🙏